spyder/plugins/findinfiles/widgets/combobox.py

```diff
@@ -44,7 +44,10 @@ class SearchInComboBox(SpyderComboBox):
     # Signals
     sig_redirect_stdio_requested = Signal(bool)
 
-    def __init__(self, external_path_history=[], parent=None, id_=None):
+    def __init__(self, external_path_history=None, parent=None, id_=None):
+        external_path_history = (
+            [] if external_path_history is None else external_path_history
+        )
```

Comment:
Please move these to be at line **77** below.

Reply:
Actually, this can just be removed as there is already a test for an empty list that works equally well for None